src/base/bittorrent/session.cpp

```diff
@@ -2193,6 +2193,8 @@ bool Session::addTorrent_impl(AddTorrentData addData, const MagnetUri &magnetUri
     p.max_connections = maxConnectionsPerTorrent();
     p.max_uploads = maxUploadsPerTorrent();
     p.save_path = Utils::Fs::toNativePath(savePath).toStdString();
+    p.upload_limit = addData.uploadLimit;
```

Comment:
These params are ignored when resume data is used, right?

Reply:
Yes, I intend these params to be effective only when adding a new torrent.